sdv/metadata/metadata.py

```diff
@@ -173,7 +173,7 @@ def set_sequence_index(self, table_name, column_name):
         self._validate_table_exists(table_name)
         self.tables[table_name].set_sequence_index(column_name)
 
-    def set_sequence_key(self, table_name, column_name):
+    def set_sequence_key(self, *args, **kwargs):
```

Comment:
I think for all of these we can just use the signature normally. For this one it would be `set_sequence_key(self, column_name, table_name)`. I don't think we have to support a different order